internal/helmdeployer/rollback.go

```diff
@@ -23,40 +25,44 @@ func (h *Helm) RemoveExternalChanges(ctx context.Context, bd *fleet.BundleDeploy
 	if err != nil {
 		return "", err
 	}
-	currentRelease, err := cfg.Releases.Last(releaseName)
+	currentRelease, err := getLastRelease(cfg.Releases, releaseName)
 	if err != nil {
 		return "", err
 	}
 
-	r := action.NewRollback(&cfg)
+	r := action.NewRollback(cfg)
+	// Set ServerSideApply to "auto" to use the same apply method as the original release
+	// If not set, defaults to empty string which causes validation error in Helm v4
+	r.ServerSideApply = "auto"
+	// WaitStrategy must be set in Helm v4 to avoid "unknown wait strategy" error
+	// HookOnlyStrategy is the default behavior (equivalent to not waiting)
+	r.WaitStrategy = kube.HookOnlyStrategy
 	r.Version = currentRelease.Version
 	r.MaxHistory = bd.Spec.Options.Helm.MaxHistory
 	if r.MaxHistory == 0 {
 		r.MaxHistory = MaxHelmHistory
 	}
-	if bd.Spec.CorrectDrift.Force {
-		r.Force = true
-	}
+	// Force field removed in Helm v4
```

Comment:
`ForceReplace` is available on rollbacks though :thinking:

Reply:
Good point, that went by the board after the first adaption. Should be fixed in my follow up commit.